builtin/providers/aws/resource_aws_opsworks_rds_db_instance.go

```diff
@@ -0,0 +1,183 @@
+package aws
+
+import (
+	"log"
+	"time"
+
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/opsworks"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsOpsworksRdsDbInstance() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsOpsworksRdsDbInstanceRegister,
+		Update: resourceAwsOpsworksRdsDbInstanceUpdate,
+		Delete: resourceAwsOpsworksRdsDbInstanceDeregister,
+		Read:   resourceAwsOpsworksRdsDbInstanceRead,
+
+		Schema: map[string]*schema.Schema{
+			"id": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"stack_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"rds_db_instance_arn": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"db_password": &schema.Schema{
+				Type:      schema.TypeString,
+				Required:  true,
+				Sensitive: true,
+			},
+			"db_user": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+		},
+	}
+}
+
+func resourceAwsOpsworksRdsDbInstanceUpdate(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*AWSClient).opsworksconn
+
+	d.Partial(true)
+
+	// @TODO if these two params force new resources, is the following necessary?
+	if d.HasChange("rds_db_instance_arn") || d.HasChange("stack_id") {
```

Comment:
We don't need this part - the ForceNew will take care of this and we can document it